### PR TITLE
feat: added program badges

### LINKED ISSUE #1400: NEW: Program Badges
program badges

### PR DESCRIPTION
**Pull request by austingardner**
## Description
Program Badges
Hey guys - I'm wondering if you could take a quick look at this. I'm not quite sure why the program badges aren't showing up, but I'm pretty sure it's some issue linking the files together. Maybe I missed something with how Jekyll works, or one of the scripts needs a little addition? 

## References
closes #1400

### CODE REVIEW

**Comment by ianmcburnie** [comment]
I can fix this up and get the icons showing in the docs page.

**Comment by ianmcburnie** [comment]
Badge icons now working. I could see the source of confusion: we no longer use the `ds4/symbols.html` & `ds6/symbols.html` files, we pull all icons from a direct reference to `ds4/icons.svg` and `ds6/icons.svg`. So I have deleted those redundant files, removed the relevant lines from the script that were generating them, and removed some commented code that wasn't helping (!).

Screen Shot 2021-05-04 at 9 13 36 PM

File: src/svg/ds6/program-badges.svg [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    </symbol>
+    <symbol id="program-badge-money-back-guarantee-zl" viewBox="0 0 48 48">
+        <defs/>
+        <g clip-path="url(#clip0)">
```
Comment (ianmcburnie) [inline src/svg/ds6/program-badges.svg:41]: Not sure if these svgs were ran through an optimizer (such as SVGOMG), but looks like we could and should do some additional cleanup.

**Comment by ianmcburnie** [comment]
Branch needs rebasing with 12.5.0

**Comment by agliga** [comment]
Screen Shot 2021-05-07 at 8 33 43 AM
Latest screenshot

File: dist/program-badge/ds4/program-badge.css [hunk 2]
```diff
@@ -0,0 +1 @@
+/* stylelint-disable no-empty-sourcer */
```
Comment (ianmcburnie) [inline dist/program-badge/ds4/program-badge.css:1]: I feel like we might as well make these available in DS4.

**Review by ianmcburnie — APPROVED** [review]

**Pull request merged**